jsonrpc2.go

```diff
@@ -365,6 +365,11 @@ func (c *Conn) Call(ctx context.Context, method string, params, result interface
 	if err != nil {
 		return err
 	}
+	defer func() {
+		for range call.done {
+			// drain
+		}
+	}()
```

Comment:
I think instead you should make the channel buffered. Otherwise if `ctx.Done()` below triggers first (ie a timeout occurs), we still end up waiting for done to respond anyways